homeassistant/components/skybell/light.py

```diff
@@ -55,3 +57,8 @@ def is_on(self) -> bool:
     def brightness(self) -> int:
         """Return the brightness of the light."""
         return int((self._device.led_intensity * 255) / 100)
+
+    @property
+    def rgb_color(self) -> tuple[int, int, int] | None:
+        """Return the rgb color value [int, int, int]."""
+        return self._device.led_rgb
```

Comment:
It looks like this device has a separate field for brightness but if you find you need to split the value later from rgb

https://github.com/Danielhiversen/flux_led/blob/43437b8cfa71310eef17047ad25b5094165d64be/flux_led/base_device.py#L1203